dfhack-config/autonick.txt

```diff
@@ -0,0 +1,161 @@
+# autonick config file
+# every line in this file that is not empty and does
+# not start with "#" will be used as a nickname by the
+# autonick script.
+
+# nicknames can be multiple words
+Toady One
+Threetoe
+
+# animals
+Mouse
+Otter
+Snake
+Owl
+Bat
+Fox
+Mole
+Cat
+Badger
+Squirrel
+Kit
+Wren
+Jay
+Crow
+Raven
+Sparrow
+
+# colours
+Flash
+Red
+Gray
+Grey
+Blue
+Shadow
+Indigo
+Jade
+Umber
+
+# planets
+Mars
+Jupiter
+Saturn
+
+# nature
+Blaze
+River
+Snow
+Bones
+Rain
+Reed
+Lake
+Briar
+Brook
+Sky
+Storm
+Clay
+Ember
+
+# trees
+Ash
+Oak
+Rowan
+Aspen
+Alder
+Apple
+Beech
+Birch
+Box
+Cedar
+Cypress
+Elder
+Elm
+Larch
+Fir
+Juniper
+Lime
+Pine
+Poplar
+Spruce
+Yew
+
+# seasons
+Spring
+Summer
+Autumn
+Winter
+
+# cardinals
+North
+South
+East
+West
+
+# jobs
+Royal
+Sage
+King
+Piper
+Poet
+Sailor
+Tanner
+Mason
+Miller
+Baker
+Archer
+Hunter
```

Comment:
We should probably avoid nicknames that can be confused with professions. Otherwise "Mason" Urist, the miller could get confused, for example, with "Miller" Urist, the mason.